appveyor.yml

```diff
@@ -1,22 +1,35 @@
+image: Visual Studio 2017
+
 environment:
+  channel: nightly
   matrix:
-  - TARGET: nightly-x86_64-pc-windows-msvc
-  - TARGET: nightly-i686-pc-windows-msvc
-  - TARGET: nightly-x86_64-pc-windows-gnu
-  - TARGET: nightly-i686-pc-windows-gnu
-  - TARGET: 1.6.0-x86_64-pc-windows-gnu
+  - TARGET: x86_64-pc-windows-msvc
+    HOST: x86_64-pc-windows-msvc
+  - TARGET: aarch64-pc-windows-msvc
+    HOST: x86_64-pc-windows-msvc
+  - TARGET: i686-pc-windows-msvc
+    HOST: i686-pc-windows-msvc
+  - TARGET: x86_64-pc-windows-gnu
+    HOST: x86_64-pc-windows-gnu
+  - TARGET: i686-pc-windows-gnu
+    HOST: i686-pc-windows-gnu
+
 install:
-  - ps: if (ls -r . -fi "*.rs" | sls "`t") { throw "Found tab character" }
-  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-${env:TARGET}.exe" -FileName "rust-install.exe"
-  - ps: .\rust-install.exe /VERYSILENT /NORESTART /DIR="C:\rust" | Out-Null
-  - ps: $env:PATH="$env:PATH;C:\rust\bin"
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init -yv --default-toolchain %channel% --default-host %HOST%
+  - set PATH=%PATH%;%USERPROFILE%\.cargo\bin;%APPVEYOR_BUILD_FOLDER%
+  - ps: $Env:PKG_CONFIG_ALLOW_CROSS=1
```

Comment:
What is this environment variable even used for? There's no pkgconfig dependencies anywhere so why would this ever apply?

Reply:
You're right, I removed it